profiler/src/ProfilerEngine/Datadog.Profiler.Native/FrameStore.cpp

```diff
@@ -930,14 +928,14 @@ std::string FrameStore::GetMethodSignature(ICorProfilerInfo4* pInfo, IMetaDataIm
     pSigBlob += CorSigUncompressData(pSigBlob, &callConv);
 
     ULONG argCount = 0;
-    ClassID* methodTypeArgs = NULL;
-    ClassID* classTypeArgs = NULL;
+    ClassID classId = 0;
     ModuleID moduleId;
     // for generic support
+    std::unique_ptr<ClassID[]> methodTypeArgs = NULL;
+    std::unique_ptr<ClassID[]> classTypeArgs = NULL;
```

Comment:
prefer using `nullptr` 
```suggestion
    std::unique_ptr<ClassID[]> methodTypeArgs = nullptr;
    std::unique_ptr<ClassID[]> classTypeArgs = nullptr;
```

Reply:
Good point